### PR TITLE
Updated mods list

### PR DESCRIPTION
**Pull request by mosemister**
Updates the mods list to api 8

## Main Changes

When it comes to 1.16.5 mod issues, i found a lot more then 1.12.2 on Github, most without solutions. Not sure if the stance on this sort of thing was to ignore the ones without a solution or that the docs weren't updated for a while or there was generally not too many 1.12.2 issues. Eitherway I have included the ones without any issues as I believe that sort of information should be out there stating `I am aware of the issue`. 

For this reason I have also included a github issue tag for all issues (except abnormal core .... as I couldnt find one -> took the discord pinned as the issue) so if there is a update to any of the incompatibilities then users can click on the github issue and see the update before the docs are updated.

Screen Shot 2023-04-21 at 13 55 56-fullpage

### CODE REVIEW

**Review by ST-DDT — COMMENTED** [review]
I like the idea with referencing the issue.
Should we sort the mods alphabetically?

**Comment by mosemister** [comment]
> I like the idea with referencing the issue. Should we sort the mods alphabetically?

Sure, i was going to but Inscrutable wanted to test some of the mods out again. I got the impression to remove the mods whereby the solution is to just update sponge, so didnt want to sort until the mod list is complete

**Comment by mosemister** [comment]
> I like the idea with referencing the issue. Should we sort the mods alphabetically?

I sorted it as SOS is coming

File: source/server/spongineer/incompatible.rst [hunk 1]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
-~~~~~~~~~~~~~~~~~~~~~~
-- Versions: All
-- Problem: Crash on startup (mixin conflict).
-- Solution: Update to Spongeforge 7.2.0 or newer.
-
-MystCraft
+Forge Permissions
+~~~~~~~~~~~~~~~~~
+
+Some mods may register there own permissions with Forge. These cannot be tracked by mods including SpongeForge due to a
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: there -> their
 commas needed:  `These cannot be tracked by mods, including SpongeForge, due to [...]`

File: source/server/spongineer/incompatible.rst [hunk 2]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+Some mods may register there own permissions with Forge. These cannot be tracked by mods including SpongeForge due to a 
+bug that was fixed in December 2021 (after forge stopped updating 1.16.5) so permissions plugins will not be able to use
+these permissions.
+
+Abnormals_core
+~~~~~~~~~~~~~~
+
+- Version: 3.3.1
+- Problem: Crash on startup
+- Solution: Use unoffical patched for Sponge version on the server. Download `Abnormals_core-1.16.5-3.3.1 <https://cdn.discordapp.com/attachments/406987481825804290/949798054117122058/abnormals_core-1.16.5-3.3.1.jar>`_
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: "Use the unofficial patched version on the server.  Download [...]"

File: source/server/spongineer/incompatible.rst [hunk 3]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+
+AlexsMobs
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: We should use the plain English name where possible, `Alex's Mobs`. (and fix underline length to match).
HOWEVER, we can tweak this one. I tested it recently, and the latest (and probably last) version for 1.16.5 works fine with Spongeforge-1.16.5-36.2.5-8.1.0-RC1313 / Forge-1.16.5-36.2.39 . The compatible mod versions are `alexsmobs-1.12.1` and the dependency `citadel-1.8.1-1.16.5`. (I updated and closed the issue).

File: source/server/spongineer/incompatible.rst [hunk 4]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+- Problem: Crashes randomly
+- Solution: None so far
+- Github issue: `Exception exiting Phase crash from AlexsMobs <https://github.com/SpongePowered/Sponge/issues/3535>`_
+
+All The Mods 6
+~~~~~~~~~~~~~~
+
+- Version: 1.8.28
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: "Latest" is a relative term. It's better if we can state a definitive working version. Whatever it was tested with should be sufficient as a minimum. So, for example, SpongeForge-RC-1313+
-> This comes up a few more times below.

File: source/server/spongineer/incompatible.rst [hunk 5]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+- Solution: None so far
+- Github issue: `Bountiful Baubles blows up SF 1.16.5 <https://github.com/SpongePowered/Sponge/issues/3646>`_
+
+Conquest Reforged
+~~~~~~~~~~~~~~~~~
+
+- Version: 5.0.7
+- Problem: Breaking a block with a snow layer in hand causes crash
+- Solution: None so far
+- Github issue: `Server crashes when creaking a block with snow layer in your hand <https://github.com/SpongePowered/Sponge/issues/3621>`_
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: creaking?
Shiver me timbers.

File: source/server/spongineer/incompatible.rst [hunk 6]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+- Solution: None so far
+- Github issue: `SpongeForge incompatibility with pehkui <https://github.com/SpongePowered/Sponge/issues/3829>`_
+
+Physics Mod
+~~~~~~~~~~~
+
+- Version: 1.3.4
+- Problem: Lag on TNT use 
+- Solution: None so far
+- Github issue: `Fatel server lag when using TNT and Physics mod <https://github.com/SpongePowered/Sponge/issues/3517>`_
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: "Fatal". <s>I updated this one too, it seems to have become worse. Latest is physics-mod-2.6.7-mc-1.16.5-forge, and it just seems to die when SpongeForge is present. </s>
This one is fixed too. Physics-mod-2.6.7-mc-1.16.5-forge and Spongeforge-RC-1313 play nice together.

File: source/server/spongineer/incompatible.rst [hunk 7]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+
+Progressive Bosses
+~~~~~~~~~~~~~~~~~~
+
+- Version 3.4.3
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Illegal State Exception with Progressive Bosses mod <https://github.com/SpongePowered/Sponge/issues/3714>`_
+
+Random patches
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: Please use Title Case

File: source/server/spongineer/incompatible.rst [hunk 8]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+~~~~~~~~~~~~~~
+
+- Version: 2.4.4
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Server crash with Randompatches <https://github.com/SpongePowered/Sponge/issues/3589>`_
+
+Tickers' Construct
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: It's called `Tinkers' Construct`. (The apostrophe is optional, according to the mod page...)

File: source/server/spongineer/incompatible.rst [hunk 9]
```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
...
+- Solution: None so far
+- Github issue: `The server can't launch with Valkyrien Skies <https://github.com/SpongePowered/Sponge/issues/3809>`_
+
+World Edit
+~~~~~~~~~~
+
+- Version: 7.2.5
+- Problem: Fatal errors after startup relating to commands
+- Solution: None so far
+- Github issue: `WorldEdit command registrar issue on server start <https://github.com/SpongePowered/Sponge/issues/3540>`_
```
Comment (Inscrutable) [inline source/server/spongineer/incompatible.rst]: There's also this issue, covering the issue with the Sponge version. Progress is happening...
https://github.com/SpongePowered/Sponge/issues/3811

**Review by Inscrutable — CHANGES_REQUESTED** [review]
Nice work. I have tested and updated a few issues, and I may try testing a few more, but this should be good for now. A few typos and stuff need correcting.
It worries me when we get Mod Packs on the list, because it needs to be narrowed down to a specific mod (or mods) that cause the problem. If we're not careful the issues might just turn out to be duplicates of others.

File: source/server/spongineer/incompatible.rst (hunk 1 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated

File: source/server/spongineer/incompatible.rst (hunk 2 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated

File: source/server/spongineer/incompatible.rst (hunk 3 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated

File: source/server/spongineer/incompatible.rst (hunk 4 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Removed ``latest``

File: source/server/spongineer/incompatible.rst (hunk 5 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Typo - fixed

File: source/server/spongineer/incompatible.rst (hunk 6 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated solution

File: source/server/spongineer/incompatible.rst (hunk 7 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated

File: source/server/spongineer/incompatible.rst (hunk 8 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Updated

File: source/server/spongineer/incompatible.rst (hunk 9 above)
Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Added issue.

Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: I have heard the plugin version is now working on Sponge Vanilla, does the plugin also work on sponge forge too?

Comment (Inscrutable) [reply source/server/spongineer/incompatible.rst]: I haven't seen (3) and (4) from issue 3811 fixed yet, so probably not.

File: source/server/spongineer/incompatible.rst (hunk 6 above)
Comment (Inscrutable) [reply source/server/spongineer/incompatible.rst]: Fatal is still spelled wrong.

Comment (mosemister) [reply source/server/spongineer/incompatible.rst]: Now its fixed

**Review by Inscrutable — APPROVED** [review]

**Pull request merged**